### DISCUSSION TITLE
Revision flagship RGB LEDs

### DISCUSSION THREAD

**Discussion by c1runner** [mathoudebine/turing-smart-screen-python#330] [Q&A] [answered]
Hello,
I don't really understand if the Rev flagship backplate LEDs are supported? My are always off and I don't see any options for LEDs in the menu. Is it possible to get them to work and how is it done? Thanks.

**Comment by ooemir23** [comment]
nice question..

**Comment by c1runner** [comment]
Additional info: The screen that I use is definitely Revision flagship. I already removed the two screws from the backplate to be sure. The 4 LEDs are there and between the backplate and the board is this clear white plastic thing. I know that a theme can have an RGB setting. But no matter which color I set the LEDs remain without light. I thought that it could be brightness related but even with 100% Brightness I have no light. I just wonder...

**Comment by mathoudebine** [comment]
Hi @c1runner
I can confirm this program does support backplate LEDs for rev. flagship. I just tested the latest release with my rev. flaghsip display and LEDs are turning on automatically when the main program is started.
Can you post here what is inside your `log.log` file in the project main folder?

**Comment by c1runner** [comment]
Hello @mathoudebine,
here it is. Thank you for your help. =)

[log.log](https://github.com/mathoudebine/turing-smart-screen-python/files/12707119/log.log)

**Reply by mathoudebine** [reply]
Thanks @c1runner , I see that your screen is correctly identified as a flagship revision:
`HW sub-revision: 2578`
so it means the System Monitor program will try to turn on the backplate LEDs.

Can you check that you have the correct format for RGB LEDs color in your theme.yaml?
It should be like this:
```yaml
---
display:
  DISPLAY_ORIENTATION: landscape  # or portrait depending on your theme
  DISPLAY_RGB_LED: 255, 0, 0
```

Did you manage to turn on and control the RGB LEDs from the `ExtendScreen.exe` Windows software that is provided with the screen?

**Reply by c1runner** [reply, marked as answer]
Hi @mathoudebine,
I have the correct format for RGB LEDs in my theme.yaml and I tried several other themes with correct values aswell. No luck. Last I tried ExtendScreen.exe (first time I used it, used your great software from the start) and still there is no light. So I guess that my smart screen is defective if there is no secret button to turn the LEDs on... =(
Thank you so much for your help and keep up the good work!

**Reply by mathoudebine** [reply]
Sorry to hear that, I'm sure you can ask for a refund or a replacement from your reseller! Have a nice day and thanks
